src/worker-helpers.js

```diff
@@ -119,8 +120,15 @@ export function getArgv(type, config, filePath, fileDir, givenConfigPath) {
   if (config.eslintRulesDir) {
     let rulesDir = resolveEnv(config.eslintRulesDir)
     if (!Path.isAbsolute(rulesDir)) {
+      const splitPath = rulesDir.split('/')
+      if (splitPath[0] === '~') {
```

Comment:
Does `untildify` only work if it _begins_ with `~`? If not shouldn't we just pass it on if it contains `~` at all?